README.md

```diff
@@ -13,17 +13,23 @@ The "National Integration Adaptor - GP2GP" implements a GP2GP 2.2 producer using
 ## Configuration
 
 The adaptor reads its configuration from environment variables. The following sections describe the environment variables
- used to configure the adaptor. 
- 
+ used to configure the adaptor.
+
 Variables without a default value and not marked optional, *MUST* be defined for the adaptor to run.
 
 ### General Configuration Options
 
-| Environment Variable               | Default                   | Description 
+| Environment Variable               | Default                   | Description
 | -----------------------------------|---------------------------|-------------
 | GP2GP_SERVER_PORT                  | 8080                      | The port on which the SCR API will run
 | GP2GP_LOGGING_LEVEL                | INFO                      | Application logging level. One of: DEBUG, INFO, WARN, ERROR. The level DEBUG **MUST NOT** be used when handling live patient data.
 | GP2GP_LOGGING_FORMAT               | (*)                       | Defines how to format log events on stdout
+| GP2GP_STORAGE_TYPE                 | LocalMock                 | Defines the storage solution being used (S3, Azure, LocalMock)
+| GP2GP_STORAGE_CONTAINER_NAME       | for-nia-testing           | Defines the name of the BlobStorage container on Azure or bucket on S3
+| AZURE_STORAGE_CONNECTION_STRING    |                           | Defines the connection string used to connect to azure blob storage
```

Comment:
Prefix with GP2GP since it is "our" variable. `GP2GP_AZURE_STORAGE_CONNECTION_STRING` they get a bit long winded but 🤷